en/docs/install-and-setup/setup/api-controller/building-jenkins-ci-cd-pipeline-apictl.md

```diff
@@ -0,0 +1,328 @@
+# Building a CI/CD Pipeline for APIs Using Jenkins
+
+In the modern digital-era, many organizations adopt digital transformation technologies to keep up with the changes. APIs are one of the basic building blocks used in a digitally-driven organization. When the number of APIs managed by the organization grows, they need to have an automated process to handle the rapid API development process. Having a proper continuous integration and continuous deployment (CI/CD) process would give an added advantage to your organization. 
+
+In this section, you will learn how to build an automated process using the WSO2 API Controller(CTL) with WSO2 API Manager 4.0.0 . You will see how a solution is built using version control system (Github), artifact repository(JFrog Artifactory) and CI/CD Tool (Jenkins).
+
+<a href="{{base_path}}/assets/img/learn/api-controller/cicd-jenkins-arch.png"><img src="{{base_path}}/assets/img/learn/api-controller/cicd-jenkins-arch.png" alt="" name="" width="600"></a>
+
+## Setting up the environment
+
+### Step 1 - Setup Jenkins
+
+Before you begin, you need to set up the Jenkins server on a remote machine which has public access. To  use webhooks in Github and Artifactory repository to trigger Jenkins jobs you need a publicly accessible Jenkins server url. If you are trying this on a local machine, you may have to use a tool such as [socketxp](https://www.socketxp.com/) to create a tunnel.
+
+In this section, the instructions are tested out on an ubuntu server setup on [https://cloud.google.com/compute](https://cloud.google.com/compute) . 
+
+Log in to the server and install following.
+
+1. Install jq  using the command given below.
+  ```
+  sudo apt-get install jq
+
+  ```
+2. Install the [apictl tool](https://github.com/wso2/product-apim-tooling/releases/tag/v4.0.0). Extract to a location henceforth referred as `CTL_HOME`.
```

Comment:
```suggestion
2. Install the [apictl tool](https://github.com/wso2/product-apim-tooling/releases/tag/v4.0.0). Extract to a location henceforth referred as `CTL_HOME`.
```
Let's not use the github URL here, since when we are doing patch releases we need to change this URL everytime. Instead shall we point to this page -> https://apim.docs.wso2.com/en/latest/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-apictl ?
We usually update this page during each patch release we do